hummingbot/strategy/pure_market_making/pass_through_filter_delegate.pyx

```diff
@@ -1,7 +1,21 @@
 from .pure_market_making_v2 cimport PureMarketMakingStrategyV2
-
+from .data_types import (ORDER_PROPOSAL_ACTION_CREATE_ORDERS,
+                         OrdersProposal)
 
 cdef class PassThroughFilterDelegate(OrderFilterDelegate):
+
+    def __init__(self, order_placing_timestamp: float):
+        super().__init__()
+        self._order_placing_timestamp = order_placing_timestamp
+
+    @property
+    def order_placing_timestamp(self) -> float:
```

Comment:
This interface is not specified in the filter delegate. Filter delegates are intended to be plugins. If you specify something in an implementation class and make the owner of the delegate (i.e. PureMarketMakingStrategyV2) use that... then you've effectively required all other filter delegate plugin to implement it.

You should either add this interface in order filter delegate s.t. it's required and handled automatically for all filter delegate plugins, or you should find a different way to implement this without breaking the interface contract.